server/schedule/operator/step.go

```diff
@@ -561,8 +569,9 @@ func (mr MergeRegion) Influence(opInfluence OpInfluence, region *core.RegionInfo
 }
 
 // Timeout returns true if the step is timeout.
-func (mr MergeRegion) Timeout(start time.Time, regionSize int64) bool {
-	return time.Since(start) > fastStepWaitDuration(regionSize)*10
+// The merge operator must wait for the first operator finished, so the executing duration must larger than add learner.
+func (mr MergeRegion) Timeout(regionSize int64) time.Duration {
+	return slowStepWaitDuration(regionSize) * 2
```

Comment:
Is the change expected?

Reply:
yes, the merge step must wait the first operator (move) finished .

Reply:
I feel that * 2 is not enough. The first operator contains multiple move peer steps